### DISCUSSION TITLE
My account was hacked and flagged.

### DISCUSSION THREAD

**Discussion by ymzc** [community/community#112098] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My another account “CraftDream” seems to be hacked.
A repository named “CraftDream1” inexplicably appeared on that account, spreading some illegal content, which resulted in my account being flagged. I didn’t create that repository. I can login, but my profile and contributions aren't visible to others. I try to contact GitHub Support but no response for more than two weeks.
I have now activated 2FA and deleted that repository which was created by the hacker.
I hope my account can be recovered soon.
I used this account to create a discussion because I have tried to use that account but failed.

**Comment by dorochadev** [comment]
Best option is to simply wait for GitHub Support to respond, not much else you can do.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @ymzc, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
